scripts/build_buildenv.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+DOCKER_IMAGE=$(grep -A1 container: .github/workflows/build.yml |grep image: | xargs | cut -d ' ' -f2)
+
+# Increment current version to avoid override
+DIGIT=$(echo "$DOCKER_IMAGE" | cut -d '.' -f2)
+DOCKER_IMAGE=$(echo "$DOCKER_IMAGE" | cut -d '.' -f1).$((DIGIT + 1))
+
+docker build --no-cache -t "$DOCKER_IMAGE" scripts
+
+# Uncomment below to push the image
+# docker push "$DOCKER_IMAGE"
```

Comment:
Is the commented code intended?

Reply:
Yes, I'd prefer that you actively make a choice about pushing the image. I'll add an echo stating that you should un-comment to make sure that it's not forgotten :+1: